lightfm/cross_validation.py

```diff
@@ -37,8 +37,8 @@ def random_train_test_split(interactions,
         The interactions to split.
     test_percentage: float, optional
         The fraction of interactions to place in the test set.
-    random_state: np.random.RandomState, optional
-        The random state used for the shuffle.
+    seed: int, optional
```

Comment:
I think the argument name should remain `random_state`, but the code should be changed to do the following:
1. If a `RandomState` instance is passed, behave exactly as before.
2. If something else is passed, construct a random state from it, treating it as a seed.

The reason is backward compatibility:
1. Uses in the wild will rely on the argument name.
2. Uses in the wild will rely on the function working with a `RandomState` instance.

An (slightly overcomplicated) example of this is [here](https://github.com/lyst/lightfm/blob/master/lightfm/lightfm.py#L201).

Reply:
That makes sense. I'll get that in there.

Reply:
I think the docs should be worded in a way that makes it clear that providing an integer as a seed is preferred over providing a np.RandomState. Is this ok with you or would you prefer to keep the language neutral?